fastai/optimizer.py

```diff
@@ -44,17 +68,23 @@ def set_hyper(self, k, v):
         assert len(v) == len(self.hypers), f"Trying to set {len(v)} values for {k} but there are {len(self.param_lists)} parameter groups."
         self._set_hyper(k, v)
 
+    def unfreeze(self): self.freeze_to(0)
     @property
     def param_groups(self): return [{**{'params': pg}, **hp} for pg,hp in zip(self.param_lists, self.hypers)]
```

Comment:
Missing docstring.

Reply:
It is only self, I think this doesn't need docment and IIRC `freeze_to` has some docments

Reply:
I think a docstring would be helpful though... It could be taken from the PyTorch docs (there's param_groups there too)

Reply:
I finally decided to copy the descriptions of `add_docs` but they are duplicated, should I delete the ones on add_docs? or all fine having them in docstring and add_docs?